dm/openapi/spec/dm.yaml

```diff
@@ -1833,3 +1846,12 @@ components:
       required:
         - "success_task_list"
         - "failed_task_list"
+    GetClusterIDResponse:
+      type: object
+      properties:
+        cluster_id:
```

Comment:
weak suggestion: maybe use string ID is better for human readning?

Reply:
> weak suggestion: maybe use string ID is better for human reading?

Random integer type is needed, so i didn't adopt uuid.